src/applications/mhv/secure-messaging/tests/fixtures/threads/message-thread-reducer.json

```diff
@@ -1,28 +1,28 @@
 {
-  "messageDetails": {
-    "message": {
-      "replyToName": "MORGUN, OLEKSII",
-      "messageId": 2715943,
-      "category": "COVID",
-      "subject": "Covid Inquiry from OM",
-      "body": "Second test reply from Clinician Oleksii",
-      "attachment": null,
-      "sentDate": "2023-03-23T14:58:03.000Z",
-      "senderId": 2708253,
-      "senderName": "MORGUN, OLEKSII ",
-      "recipientId": 251391,
-      "recipientName": "MHVDAYMARK, MARK ",
-      "readReceipt": "READ",
-      "triageGroupName": "SM_TO_VA_GOV_TRIAGE_GROUP_TEST",
-      "proxySenderName": null,
-      "threadId": 2710544,
-      "folderId": 0,
-      "messageBody": "Second test reply from Clinician Oleksii",
-      "draftDate": null,
-      "toDate": null,
-      "hasAttachments": false
-    },
-    "messageHistory": [
+  "threadDetails": {
+    "messages": [
+      {
+        "replyToName": "MORGUN, OLEKSII",
+        "messageId": 2715943,
+        "category": "COVID",
+        "subject": "Covid Inquiry from OM",
+        "body": "Second test reply from Clinician Oleksii",
+        "attachment": null,
+        "sentDate": "2023-03-23T14:58:03.000Z",
+        "senderId": 2708253,
+        "senderName": "MORGUN, OLEKSII ",
```

Comment:
Aren't we avoiding using pii for mock data?

Reply:
Yes, thank you for catching this. I will replace with mock data